src/Altinn.JwkGenerator/version.txt

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
If the goal is to use the release-system I've made for the auth-utils repo, you should remove the manually created changelog, and set version to 0.0.0. The first release PR will update these to v1.